scio-core/src/main/scala/com/spotify/scio/VersionUtil.scala

```diff
@@ -99,7 +99,7 @@ private[scio] object VersionUtil {
 
   def checkVersion(
     current: String,
-    latest: Option[String],
```

Comment:
making this a call-by-name would also work `latest: => Option[String]`

Reply:
That was my initial thought but feels it could be confusing, and the argument is really only used for testing.